### DISCUSSION TITLE
Emby Security Cams plugin?

### DISCUSSION THREAD

**Discussion by thereal-BLKMGK** [mrlt8/docker-wyze-bridge#1366] [Q&A]
Emby has a plugin named Security Cams - it pretty much does what you would expect, allows you to setup a security camera as a "channel" in Emby. However to date I've not been able to get it working with any of my Wyze cameras. I suspect it's simply the way I'm configuring it and am wondering if anyone has been able to get this working? Just seems like it could be a pretty cool little trick for live streaming a camera feed to others if I was able to get it working :-)

**Comment by mrlt8** [comment]
hmm, could you try:
```
http://<bridge-ip>:5000/cams.m3u8
```

**Reply by thereal-BLKMGK** [reply]
If only it were that easy! The plugin asks for individual cameras and builds it's own .m3u8. It allows for a large number of protocols including multiple protocols that your software supports. I tried pointing it at one camera using the bridge URL and selected every protocol in turn and saved the various items as cameras. Did the required "refresh" which wrote the m3u8 and then attempted to access them - none worked and it claims it cannot find a compatible stream. Frustratingly the m3u8 doesn't look THAT different than the one your page builds when I download the URL you provided and VLC opens both the generated file and the downloaded file just fine. 

This is where the plugin is coming from https://emby.media/community/index.php?/topic/116062-security-cams-channel-plugin/#comment-1223647

Tried using the RTSP format similar to what they display but copied from your software's status page which seems more like the way to go. Something like "rtsp://IPADDRESS:8554/gate" where gate is name of camera. No dice. However I can take the string that it creates in the m3u8 file, paste it into VLC as a new stream, and it pops up instantly - argh! 

As a last desperate attempt I also have a Wyze cam with Thingino on it and the following string actually WORKED "rtsp://password:password@ipaddress/ch0" If I remove the ch0 it fails - shocker, note I don't specify the port as it's default.  I'm fairly confused. I record both of these cameras in BlueIris. The config for the camera leveraging your software looks much the same in BlueIris other than the RTSP port but I've put that in the URL for this plugin to connect and the stream is the name of the camera just like I tried.

Giving up for the night. One last thing though - I don't see anything in the logs when I try to reach out to your software using this plugin. Emby is in a container, wondering if there's a port issue but it seems to have no issue going to the default RTSP port and I've done nothing special to allow that either...

**Reply by mrlt8** [reply]
Can you try the m3u8 from the bridge directly in M3U TV Tuner? You might also want to see if your RTSP stream is accessible from another device on your network.
